.circleci/config.yml

```diff
@@ -379,8 +385,12 @@ workflows:
       - cpp-gcc:
           matrix:
```

Comment:
```suggestion
      - cpp-gcc:
          name: cpp-gcc-<< matrix.gcc-version >>-<< matrix.cpp-standard >>
          matrix:
```
otherwise it looks like
<img width="480" height="270" alt="image" src="https://github.com/user-attachments/assets/cef778af-ea7c-4b64-ba01-104ffbfa7dc0" />